service/lightsail/serializers.go

```diff
@@ -656,6 +656,15 @@ func (m *awsAwsjson11_serializeOpCreateContainerServiceRegistryLogin) HandleSeri
 	httpBindingEncoder.SetHeader("Content-Type").String("application/x-amz-json-1.1")
 	httpBindingEncoder.SetHeader("X-Amz-Target").String("Lightsail_20161128.CreateContainerServiceRegistryLogin")
 
+	jsonEncoder := smithyjson.NewEncoder()
+	if err := awsAwsjson11_serializeOpDocumentCreateContainerServiceRegistryLoginInput(input, jsonEncoder.Value); err != nil {
+		return out, metadata, &smithy.SerializationError{Err: err}
+	}
+
+	if request, err = request.SetStream(bytes.NewReader(jsonEncoder.Bytes())); err != nil {
+		return out, metadata, &smithy.SerializationError{Err: err}
+	}
+
```

Comment:
If there is no input, why do we need to generate these helpers?

Reply:
Posted PR too soon, I'm going back and regenerating the clients to fix this. The rule should do nothing if stub synthetic clone, else do what ever serialization is needed. Don't have short cut behavior of `{}`.   Will post update shortly fixing this.